perceval/simulators/simulator.py

```diff
@@ -398,3 +396,17 @@ def evolve(self, input_state: Union[BasicState, StateVector]) -> StateVector:
 
         result_sv.normalize()
         return result_sv
+
+    def evolve_svd(self, svd: Union[SVDistribution, StateVector, BasicState]) -> SVDistribution:
+        """Compute the SVDistribution evolved through a Linear Optical circuit"""
+        if not isinstance(svd, SVDistribution):
+            return SVDistribution(self.evolve(svd))
+
+        # If it's actually an SVD
+
+        new_svd = SVDistribution()
+        for sv, p in svd.items():
+            new_sv = self.evolve(sv)
+            new_svd[new_sv] = p
+
+        return new_svd
```

Comment:
In order to be consistent with `probs_svd` call, this one should support post-selection too.
(We just discussed this orally, sorry, it's not in the Jira ticket, sorry)